community/modules/file-system/nfs-server/scripts/install-nfs-server.sh.tpl

```diff
@@ -12,15 +12,22 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+set -ex
+
+# format, add to fstab, and mount the disk. See https://cloud.google.com/compute/docs/disks/add-persistent-disk
+uuid=$(uuidgen)
+mkfs.ext4 -F -m 0 -U "$uuid" -E lazy_itable_init=0,lazy_journal_init=0,discard /dev/disk/by-id/google-attached_disk
```

Comment:
won't this still format the device if you reboot the machine? how do you prevent the startup script from wiping the data on the second reboot?

Reply:
Addressed, added if-condition